.github/workflows/unit-tests-diff-processor.yml

```diff
@@ -40,3 +40,5 @@ jobs:
         run: |
           cd tools/diff-processor
           go test -v ./...
+        env:
```

Comment:
Could you add this to the TPG section as well? I know it's not strictly required for this change but it would be nice to do anyway (since it should pass in both cases.)